ansible/roles/collectory/tasks/vm-tasks.yml

```diff
@@ -18,21 +18,75 @@
     - db
     - collectory
 
+
+#deploy as systemd service
+- name: add collectory jar and setup systemd service
+  include_role:
+    name: exec-jar
+  vars:
+    service_name: "{{collectory_app | default('collectory')}}"
+    use_openjdk17: "{{ collectory_version | string is version('5.2.0', '>') }}"
+    log_config_filename: "{{ logback_config_filename }}"
+    java_headless: True
+  tags:
+    - deploy
+    - service
+    - collectory
+  when: exec_jar | default(false) | bool
+
+- name: Extract assets (images) from jar for systemd service
+  block:
+    - name: Extract assets from jar
+      command: "jar xf /opt/atlas/{{ collectory_app }}/{{ collectory_app }}.jar {{ item }}"
+      loop:
+        - "assets/map/marker-icon.png"
+        - "assets/map/marker-icon-2x.png"
+        - "assets/map/marker-icon-shadow.png"
+      args:
+        chdir: "/opt/atlas/{{ collectory_app }}"
+
+    - name: copy asserts folder into static/images
+      shell: |
+        mkdir -p /opt/atlas/{{ collectory_app }}/static/images
+        cp -R /opt/atlas/{{ collectory_app }}/assets/map /opt/atlas/{{ collectory_app }}/static/images/
+
+    - name: Change owner of /opt/atlas/{{ collectory_app }}/static/images to {{ collectory_service_system_user}}
+      file:
+        path: "/opt/atlas/{{ collectory_app }}/static/      "
+        owner: "{{ collectory_service_system_user }}"
+        group: "{{ collectory_service_system_user }}"
+        recurse: yes
+      notify:
+        - restart collectory
+  when: exec_jar | default(false) | bool
+
+- name: set log file for collectory systemd service
+  file: path="/var/log/collectory" owner="{{ data_owner_user | default('collectory') }}"  state="directory" group="{{ data_owner_user | default('collectory') }}" recurse=true
+  notify:
+    - restart collectory
+  tags:
+    - collectory
+  when: exec_jar | default(false) | bool
+
+#deploy on tomcat
 - name: set data ownership [all data is owned by tomcat]
   file: path={{ collectory_data_dir }} owner={{ tomcat_user }} group={{ tomcat_user }} recurse=true
   notify:
     - restart tomcat
   tags:
     - properties
     - collectory
+  when: not (exec_jar | default(false) | bool)
 
 - include_tasks: ../../tomcat_deploy/tasks/main.yml
   vars:
-    war_url: '{{ collectory_war_url }}'
+    #only used when tomcat_deploy_from_url
+    war_url: '{{ collectory_war_url | default('')}}'
```

Comment:
The `collectory_war_url` was previously set in `vars/main.yml` as 
`collectory_war_url: "{{ collectory_maven_repo_ws_url | default(maven_repo_ws_url) }}"`

Please revert to the prior default value instead of `empty string`.